Docker.DotNet.sln

```diff
@@ -1,39 +1,93 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio 14
-VisualStudioVersion = 14.0.25123.0
+# Visual Studio 15

```

Comment:
I don't love requiring VS 15 but oh well. We can still use the command line I guess